website/source/api/agent/service.html.md

```diff
@@ -495,14 +495,16 @@ Note that this endpoint, unlike most also [supports `snake_case`](/docs/agent/se
 service definition keys for compatibility with the config file format.
 
 - `Name` `(string: <required>)` - Specifies the logical name of the service.
-  Many service instances may share the same logical service name.
+  Many service instances may share the same logical service name. We recommend using
+  [valid DNS labels](https://en.wikipedia.org/wiki/Hostname#Restrictions_on_valid_hostnames)
+  to avoid DNS issues.
```

Comment:
Would it be fair to say that we recommend valid DNS labels so that the service names are compatible with any external DNS? Or that is its industry best practice? Overall, the callout is great! I'm just wondering if we can be slightly more descriptive about "issues" while still being concise.

Reply:
Yeah good call, I was reaching for a better phrase to use here, so maybe replace:

> to avoid DNS issues

with

> for compatibility with external DNS

Sounds good to me. @mkeeler does that sound right technically?